scripts/gen_dictionaries.py

```diff
@@ -182,7 +182,7 @@ def GetFooter():
             category = type.get('category')
             if category == 'struct':
                 name = type.get('name')
-                print('found type: ' +name)
+                #print('found type: ' +name)
```

Comment:
This change seems unrelated.

Reply:
Yes, this removes some debug output leftover from https://github.com/KhronosGroup/OpenCL-Docs/pull/298 that I noticed while making these changes.

Reply:
Thanks. I just wanted to check that this change was intentional.